easybuild/easyconfigs/p/protozero/protozero-1.8.1-foss-2025b.eb

```
@@ -0,0 +1,29 @@
+easyblock = "CMakeMake"
+
+name = 'protozero'
+version = '1.8.1'
+
+homepage = "https://github.com/mapbox/protozero/"
+description = "Minimalistic protocol buffer decoder and encoder in C++."
+
+toolchain = {'name': 'foss', 'version': '2025b'}
+
+source_urls = ['https://github.com/mapbox/protozero/archive/']
+sources = [{
+    'download_filename': 'v%(version)s.tar.gz',
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = ['6c7a896f1dc08435e8cd4f3780ff688cd0bfce6890599b755f6f3cb36398dc25']
+
+builddependencies = [('CMake', '4.0.3')]
+
+runtest = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['include/%(name)s'],
+}
+
+modextrapaths = {'MODULE_LOAD_ENV_HEADERS': 'include/%(name)s'}
```

```suggestion
modextrapaths = {MODULE_LOAD_ENV_HEADERS: 'include/%(name)s'}
```

This already is a string constant, so no need to make it a string 😄